pkg/controller/stacks/install/installjob.go

```diff
@@ -247,7 +247,17 @@ func (jc *stackInstallJobCompleter) createJobOutputObject(ctx context.Context, o
 			obj.SetNamespace(ns)
 		}
 
+		// The package isn't *quite* what we will use as the image
+		pkg := i.GetPackage()
+		stackImg, err := i.ImageWithSource(pkg)
```

Comment:
Do we really need to `ImageWithSource` here? I think this is redundant to the `controllerImageSourcer` step a few lines below.

Reply:
It is possible that `GetPackage` is empty (unrealistic now, but was the intent for crd-lookup stackinstalls).  I don't know if this is the first place that would be making the assertion that all StackInstalls include a `package`, but we may want to avoid pushing the `image` deeper into the controller spec if the `package` is empty.

This may be undefined behavior that happens to result in the same outcome, so perhaps there is nothing to really do about this, now.

Reply:
Good point - this does seem redundant. I can't remember why I originally felt like this was needed now. I'll make a couple adjustments.

Reply:
This is removed now; resolving